packages/shared/src/components/footer/FooterLinks.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import { contentGuidelines, webappUrl } from '../../lib/constants';
+import { withExperiment } from '../withExperiment';
+import { feature } from '../../lib/featureManagement';
+
+export type FooterLinksProps = {
+  className?: string;
+};
+
+export const FooterLinks = withExperiment(
+  ({ className }: FooterLinksProps): ReactElement => {
+    return (
+      <ul
+        className={classNames(
+          className,
+          'm-0 flex flex-row flex-wrap justify-center gap-3 p-0 text-text-tertiary typo-caption1',
+        )}
+      >
+        <li>&copy; 2024 Daily Dev Ltd.</li>
+        <li>
+          <a href={contentGuidelines}>Guidelines</a>
```

Comment:
Think it should link external right this one?

Reply:
`_blank`?

Reply:
Yeah would think so as it goes outside webapp.
so _blank + rel defaults